src/components/DropDownMenu/DropDownMenu.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { memo } from 'react';
+import cn from 'classnames';
+import { Person } from '../../types/Person';
+
+type Props = {
+  visiblePeople: Person[];
+  onSelect: (
+    event: React.MouseEvent<HTMLAnchorElement, MouseEvent>,
+    personSlug: string,
+    personName: string,
+    status: boolean,
+  ) => void
+};
+
+export const DropDownMenu:React.FC<Props> = memo(({
+  visiblePeople,
+  onSelect,
+}) => {
+  return (
+    <div className="dropdown-menu" role="menu">
+      <div className="dropdown-content">
+        {visiblePeople.length !== 0
+          ? (
```

Comment:
In my opinion this syntax looks better 
```suggestion
        {visiblePeople.length ? (
```